.github/workflows/ci.yml

```diff
@@ -57,7 +57,13 @@ jobs:
 
           output_file="scripts/_ci_dependencies.ash"
           while read -r line || [ -n "$line" ]; do
-            echo "cli_execute('svn checkout ${line}');" >> "$output_file"
+            if [[ "$line" == *".git"* ]]; then
```

Comment:
This is for my own knowledge. Why is this change needed for autoscend but not Garbo? Didn't see a similar change in your Garbo pr

Reply:
garbo doesn't validate anything through mafia. this is specifically there to install dependencies so mafia can run and validate autoscend.ash